plugins/mergeStyles.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const { querySelectorAll, closestByName } = require('../lib/xast.js');
+const { getCssStr,        setCssStr     } = require('../lib/css-tools');
```

Comment:
You can install prettier in your ide.

Reply:
So these extra spaces for padding should be rather avoided?